packages/hoppscotch-selfhost-web/package.json

```diff
@@ -23,62 +23,58 @@
     "postinstall": "pnpm run gql-codegen"
   },
   "dependencies": {
-    "@fontsource-variable/inter": "^5.0.8",
-    "@fontsource-variable/material-symbols-rounded": "^5.0.7",
-    "@fontsource-variable/roboto-mono": "^5.0.9",
```

Comment:
Aren't these font packages required by `unplugin-fonts` anymore?

refer: https://github.com/cssninjaStudio/unplugin-fonts#fontsource

Reply:
i tested it and the fonts worked fine. 

but digging a little in it seems our app was not using Unfont generated font css. we were directly importing the font files from the fontsource packages.

in `hoppscotch-common/src/index.ts`, we have direct imports from the fontsource packages. in order to use the unfont generated one, we might have to replace this with 'unfonts.css'

```diff
- import "@fontsource-variable/inter"
- import "@fontsource-variable/material-symbols-rounded"
- import "@fontsource-variable/roboto-mono"
+ import "unfonts.css"
```

this way we can remove these packages from `hoppscotch-common` and keep them in font-source packages only in `hoppscotch-selfhost-web` because that's where the vite config picks up packages.

let me know what you think.

Reply:
@amk-dev can you verify once again al packages works if you only add font deps in common?

Reply:
the unfonts usage is fixed. the @fontsource deps are removed from `hoppscotch-common`. but added back to `hoppscotch-selfhost-web`